portal/views/extend_flask_user.py

```diff
@@ -51,15 +51,19 @@ class LockoutLoginForm(LoginForm):
     def validate(self):
         """prevent locked out users from logging in
 
-        Before verifying the user's credentials
-        check to see if the user is locked out.
-        If the user is locked out display an error
+        If user has exceeded failed attempts, display an error
         message below the password field.
+
         """
+        success = super(LockoutLoginForm, self).validate()
```

Comment:
This can fail for reasons other than a password verification failure. See https://github.com/uwcirg/Flask-User/blob/v0.6.21.1/flask_user/forms.py#L192

Reply:
Took another look through this code and think we're safe. Thanks for making this change.